l10n/en/privacy/index.ftl

```diff
@@ -39,13 +39,8 @@ privacy-index-read-more-about-our-ongoing = Read more about our ongoing privacy
 privacy-index-outdated-policies = Outdated Policies
 privacy-index-mozilla-websites-communications = { -brand-name-mozilla } Websites, Communications & Cookies
 privacy-index-firefox-browser = { -brand-name-firefox-browser }
-privacy-index-firefox-os = { -brand-name-firefox-os }
-privacy-index-firefox-private-network = { -brand-name-firefox-private-network }
-privacy-index-firefox-reality = { -brand-name-firefox-reality }
 privacy-index-firefox-focus = { -brand-name-firefox-focus }
 privacy-index-thunderbird = { -brand-name-thunderbird }
-privacy-index-firefox-better-web = { -brand-name-firefox-better-web }
-privacy-index-firefox-fire-tv = { -brand-name-firefox } for { -brand-name-fire-tv }
```

Comment:
Please check if any of these brand terms are still used. If not, they should be removed also from `brands.ftl`.

P.S. Why in the world do we still have Firefox OS newsletter references?